maxz.js

```diff
@@ -1,6 +1,12 @@
 module.exports = function maxz (els) {
   els = els || document.body.getElementsByTagName('*')
-  return Math.max.apply(Math, [].map.call(els, getZzz))
+  var indexes = [0]
```

Comment:
I hate to be that guy, but `indices`.

![me](http://img.memecdn.com/grammar-nazis-start-crying-when-they-see-your-posts_o_334445.gif)